etc/profile-m-z/neomutt.profile

```diff
@@ -50,31 +50,11 @@ include disable-programs.inc
 include disable-xdg.inc
 
 mkdir ${HOME}/.Mail
-mkdir ${HOME}/.bogofilter
-mkdir ${HOME}/.config/mutt
-mkdir ${HOME}/.config/nano
-mkdir ${HOME}/.config/neomutt
-mkdir ${HOME}/.elinks
-mkdir ${HOME}/.emacs.d
-mkdir ${HOME}/.gnupg
 mkdir ${HOME}/.mail
-mkdir ${HOME}/.mutt
-mkdir ${HOME}/.neomutt
-mkdir ${HOME}/.vim
-mkdir ${HOME}/.w3m
 mkdir ${HOME}/Mail
 mkdir ${HOME}/mail
 mkdir ${HOME}/postponed
 mkdir ${HOME}/sent
-mkfile ${HOME}/.emacs
-mkfile ${HOME}/.mailcap
-mkfile ${HOME}/.msmtprc
-mkfile ${HOME}/.muttrc
-mkfile ${HOME}/.nanorc
-mkfile ${HOME}/.neomuttrc
```

Comment:
Who is `neomuttrc` supposed to be edited? Is there something like an "Open config file" options?

Reply:
> Who is neomuttrc supposed to be edited?

Usually it's created manually with a text editor.

> Is there something like an "Open config file" options?

Not that I know of. I think if you're very proficient you might figure out a way to possibly open neovim and then edit the config file manually, but that's _very_ unorthodox. I can't imagine someone doing this on a first run when no config file exists.